pkgs/top-level/python-packages.nix

```diff
@@ -6331,7 +6331,9 @@ in {
 
   python-lzo = callPackage ../development/python-modules/python-lzo { inherit (pkgs) lzo; };
 
-  python_magic = callPackage ../development/python-modules/python-magic { };
+  python-magic = callPackage ../development/python-modules/python-magic { };
+  python_magic = self.python-magic; # Old attr, 2021-02-10
```

Comment:
Is this used anywhere in nixpkgs? If not we might change it without an alias.

Reply:
Is used by those packages:

```
pkgs/applications/networking/irc/weechat/scripts/weechat-matrix/default.nix
pkgs/development/python-modules/alot/default.nix
pkgs/development/python-modules/androguard/default.nix
pkgs/development/python-modules/beancount/default.nix
pkgs/development/python-modules/django-versatileimagefield/default.nix
pkgs/development/python-modules/mocket/default.nix
pkgs/development/python-modules/pushbullet/default.nix
pkgs/development/python-modules/pysnow/default.nix
pkgs/development/python-modules/relatorio/default.nix
pkgs/development/python-modules/thumbor/default.nix
pkgs/servers/mautrix-telegram/default.nix
pkgs/servers/pinnwand/steck.nix
pkgs/tools/misc/diffoscope/default.nix
pkgs/tools/networking/s3cmd/default.nix
```